### ISSUE TITLE
Enable multi-year memberships

### ISSUE THREAD

**Issue by Drosophilic** [biocuration/isb-website#26]
For those lucky people replete with grants to afford multiyear subscriptions.

**Comment by sandraorchard** [comment]
Will need intelligent link to Renewals, so we don't them irritate them with continual notification to renew, until it is actually due.

**Comment by zhangzhangcn** [comment]
badly needed, multiple-year membership payment

**Comment by carighi** [comment]
Ceci: 1+

**Comment by monicacecilia** [comment]
+ from @selewis

**Comment by andrewsu** [comment]
There are two settings in the PMP Pro configuration, one for "recurring subscription" and "membership expiration". Currently, we have the first one unchecked and the second one checked (set to one year).
<img width="784" alt="pmp" src="https://user-images.githubusercontent.com/2635409/30351906-5fbc1454-97d2-11e7-86c9-1935c53d6b3f.png">

I *think* if we switch those two checkboxes, we will get the behavior we all want.  But this needs to be tested.  If anyone wants to play around with this on the staging site to confirm, please have at it...  Things that I think need to be confirmed are:

- [ ] Users can easily cancel their recurring subscription
- [ ] Users get renewal emails prior to auto-charging

anything else?

**Comment by monicacecilia** [comment]
Yes, we need to test. 

I am wondering what happens with the changes. 
I suspect (like you) that ticking the 'Check if this level has a recurring subscription payment' box alone will not necessarily help with switching a payment for up to five years (for example). Also, it may only mean that members are allowed to set up a recurrent payment with their account - an automatic withdraw from whichever account they choose, not necessarily a change on how many years worth of membership they can pay. (maybe?)

I am thinking that perhaps the box labeled as 'Check this to set when membership access expires'  will also not be very helpful here. 

Basically we would need something that allows us to mark that the membership access expires after 12 months, unless you choose to pay for more months (say 60 months), in which case the expiration date has to move - and you also get a discount. 

err... that's a lot of ifs. head spinning. @andrewsu do I have access to staging site?

**Comment by monicacecilia** [comment]
Oh, we could offer a discounted 5-year version of each of the current category and modify the expiration date for those discounted categories. We'd need to figure out how to work that out on WP...

**Comment by andrewsu** [comment]
@monicacecilia yeah, you're right, the changes I proposed don't exactly address the issue of multi-year memberships.  Though still would be a useful step forward IMHO.  

If you want to play: 
Staging site: https://biocuration.staging.wpengine.com/
Login: https://biocuration.staging.wpengine.com/wp-admin
